content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -567,3 +567,95 @@ goods (i.e :guilabel:`Opening Stock` + purchases during the period - :guilabel:`
 
    .. image:: india/profit-and-loss-report.png
       :alt: Profit and Loss report
+
+.. _india/tds-tcs-threshold:
+
+TDS/TCS threshold alert
+=======================
+
+:guilabel:`TDS (Tax Deducted at Source)` and :guilabel:`TCS (Tax Collected at Source)` are tax
+provisions under Indian law, triggered when transaction amounts exceed specified thresholds.
+This alert notifies users when the value of invoices or bills surpasses these limits, prompting
+the application of the appropriate TDS/TCS.
```

Comment:
```suggestion
:abbr:`TDS (tax deducted at source)` and :abbrl:`TCS (tax collected at source)` are tax provisions
under Indian law, triggered when transaction amounts exceed specified thresholds. This alert
notifies users when the value of invoices or bills surpasses these limits, prompting the application
of the appropriate TDS/TCS.
```
Sorry if I was unclear about the GUI labels, but it should be clear in the documentation guidelines - GUI labels are just for when you're instructing the user to do something with an interactive element within Odoo, like clicking a smart button or completing a field. In that case, the text that you see in Odoo is what would get the GUI label. Let's use the abbr markup for these and follow the guidelines on capitalization.